src/subresource-signing.js

```diff
@@ -39,8 +54,10 @@ var srs = {
         /**
          * Assume the contents is valid, and attach the script
          * to the document so it runs.
+         *
+         * @param contents
          */
-        attach_script: function srs_attach_script(contents) {
+        attach_script: function(contents) {
```

Comment:
The way I see it, the burden of proof of necessity is on the one who made the most recent change ;)